gdbinit.py

```diff
@@ -34,7 +35,20 @@ def calculate_hash(file_path):
 
 
 def run_poetry_install(dev=False):
-    command = ["poetry", "install"]
+    poetry_path = shutil.which("poetry")
+    if poetry_path is None:
+        # Additional check: Look for poetry at $HOME/.local/bin/poetry
+        home_poetry_path = os.path.expanduser("~/.local/bin/poetry")
+        if os.path.exists(home_poetry_path):
+            print(f"Found Poetry at {home_poetry_path}")
+            poetry_path = home_poetry_path
+        else:
+            print(
+                "Poetry was not found on the $PATH. Please ensure it is installed and on the path, or run `./setup.sh` to update Python dependencies."
+            )
+            return 1
```

Comment:
```suggestion
            return "", "", 1
```
Since below you're returning three things, you need to return three things here too.